pyproject.toml

```diff
@@ -45,27 +42,73 @@ classifiers = [
   "Operating System :: MacOS",
 ]
 dependencies = [
-  "numpy>=1.24",
+  "click>=8.0.1",
+  "colorspacious>=1.1.2",
+  "importlib-metadata",
+  "iohub==0.1.0",
+  "ipywidgets>=7.5.1",
   "matplotlib>=3.1.1",
-  "scipy>=1.3.0",
+  "natsort>=7.1.1",
+  "numpy>=1.24",
+  "ome-zarr==0.8.3", # unpin when resolved: https://github.com/ome/napari-ome-zarr/issues/111
+  "psutil",
+  "pyqtgraph>=0.12.3",
+  "pydantic==1.10.19",
   "pywavelets>=1.1.1",
-  "ipywidgets>=7.5.1",
+  "scipy>=1.3.0",
+  "submitit",
   "torch>=2.4.1",
```

Comment:
@ziw-liu this is a new constraint for `recOrder`, and it's introducing installation issues on my local mac. 

Currently `pip install torch` on my mac installs `torch==2.2.2`, the last version that supports `x86_64`. Although `arch` gives `arm64`, `python -c "import platform; print(platform.machine())"` gives `x86_64`. 

Any advice? Does your machine have the same issue?

Reply:
Related: we added this torch constraint to unpin numpy https://github.com/mehta-lab/waveorder/commit/a7ac036e4d42278873b7fe2493b91d465cb983d2

Reply:
I think you are using Rosetta-emulated Conda/Python that we used to install for compatibility with Micro-Manager. Check `conda info | grep platform`.

Reply:
While Apple Silicon support from MM is on the roadmap (https://github.com/micro-manager/micro-manager/issues/2001) I don't think it's reasonable to expect it to come any time soon. My vote would be to not support MM on macOS in waveorder due to this conflict, since it's not going to be viable in production anyway (due to lack of drivers).

Reply:
Thanks @ziw-liu. Installation worked well with an `arm-64` conda. 

I agree that we do not need to support MM on macOS.